feature-libs/product-configurator/rulebased/cpq/cpq-configurator-normalizer.spec.ts

```diff
@@ -108,6 +109,7 @@ const cpqAttribute2: Cpq.Attribute = {
   description: cpqAttributeDescription2,
   label: cpqAttributeLabel2,
   displayAs: cpqAttributeDisplayAs2,
+  dataType: cpqAttributeDataType2,
```

Comment:
In my opinion it does not make sense to define a constant if it is used exactly once. Also the name 'cpqAttributeDataType2' is not very descriptive. I would prefer to inline this constant, as well as all other constants in the file that are used only once.

Reply:
The const name cpqAttributeDataType2 means that it is used in the attribute2 rather than a 'special' data type. I agree, that it is better to not introduce this const and to inline the related value in the attribute definition directly.  Such constants are removed now.